client/src/components/App.jsx

```diff
@@ -32,7 +32,7 @@ export default (props) => {
         <Overview />
         <RelatedItems />
         <QuestionAnswer />
-        <RatingsReviews />
+        <RatingsReviews productId={product}/>
```

Comment:
In general I wouldn't pass the product ID in as a prop at this stage. That should definitely be accessed by the useSelector hook in ReviewList.jsx components.

```suggestion
        <RatingsReviews />
```